recipe/meta.yaml

```diff
@@ -19,21 +19,21 @@ source:
     - h5py_win_setup_build.py.patch
 
 build:
-  number: 2
+  number: 3
 
 requirements:
   build:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
     - cython
     - six
     - pkgconfig
 
   run:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
```

Comment:
I absolutely can, but is that necessary? Does libhdf5 really break its ABI at such a high frequency that we need the *fourth* version number (I don't even know what to call that! 😄 ).

I want to avoid dependency hell, but I also want to avoid pinned to death hell too...